packages/demo/src/app/home/home.component.html

```diff
@@ -314,7 +314,9 @@ <h4 class="h5 bold">Automatic Migration ⚙️</h4>
                 appCopyToClipboard
                 class="d-block mt-1 p-3"
                 [languages]="['bash']"
-                highlight="npx ng update @swisspost/design-system-migrations --from=5 --to=6 --migrate-only"
+                highlight="npm install @swisspost/design-system-migrations
```

Comment:
Did you try to use npx to have temporarily the package on the system rather than install/uninstall it?

Reply:
Yes, I tried `npx -p @swisspost/design-system-migrations ng update @swisspost/design-system-migrations --from=5 --to=6 --migrate-only --allow-dirty`.
With this command the migration package is temporarily installed yet Angular CLI cannot find it.

Reply:
Nope, then let's go like this.